### PR TITLE
use yosay & this.log instead of console.log.

### PR DESCRIPTION
**Pull request by stephenplusplus**
This look right?

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -39,8 +40,8 @@ AppGenerator.prototype.askFor = function askFor() {
 
   // welcome message
   if (!this.options['skip-welcome-message']) {
-    console.log(this.yeoman);
-    console.log(chalk.magenta('Out of the box I include HTML5 Boilerplate, jQuery, and a Gruntfile.js to build your app.'));
+    this.log(yosay());
+    this.log(chalk.magenta('Out of the box I include HTML5 Boilerplate, jQuery, and a Gruntfile.js to build your app.'));
```
Comment (eddiemonge) [inline app/index.js]: i feel like this line should be part of yosay as well

**Comment by sindresorhus** [comment]
Merge conflict

**Comment by addyosmani** [comment]
Could you resolve the merge conflicts? Otherwise looks good.

**Comment by stephenplusplus** [comment]
:+1:

**Comment by sindresorhus** [comment]
Awesome. Thanks for doing the module @stephenplusplus :)

**Pull request merged**